### DISCUSSION TITLE
Terminal connection enhancements

### DISCUSSION THREAD

**Discussion by D4V3M0NK** [Ylianst/MeshCentral#5306] [Ideas]
To the great Gods of MeshCentral ... permit me to offer some useful enhancements to the Terminal feature?

Is it possible to have configured setting that allows the Terminal to periodically send a NO-OP packet to the remote system (much like SSH's ServerAliveNO-OP protocol?). I see that there's an "ssh" configuration setting, but that sets up the web based SSH client. It looks like this feature could be included in the `terminal` section of the config?

Additionally, it appears that the history of the terminal isn't included when connecting - is that also possible to add?

Thank you!!

D

**Comment by bbrendon** [comment]
Not likely. This project is mostly just in security maintenance mode. I don't think you'll see features until someone decides to take it over.

Maybe you missed the announcements. https://github.com/Ylianst/MeshCentral/discussions/4795

**Comment by JSuenram** [comment]
> Not likely. This project is mostly just in security maintenance mode. I don't think you'll see features until someone decides to take it over.
> 
> Maybe you missed the announcements. #4795
There is still progress.... don't paint it to black....
